opentracing-spring-cloud-starter-jaeger/README.md

```diff
@@ -0,0 +1,86 @@
+## Configuration options
```

Comment:
top level header is missing

Reply:
I'm not sure I follow :(

Reply:
Something like
# Jaeger OpenTracing Spring Boot starter
This starter....